src/config.rs

```diff
@@ -11,6 +11,9 @@ use crate::dimension::Dimension;
 use crate::server::{NewClientData, Server, SharedServer};
 use crate::text::Text;
 use crate::{Ticks, STANDARD_TPS};
+use crate::protocol::{Node, VarInt};
+use crate::protocol::node::{Literal, NodeData};
+use crate::protocol::packets::s2c::play::Commands;
```

Comment:
Stuff from the `protocol` module should not be in the public API unless it's being reexported. The protocol module is `#[doc(hidden)]` and might be made private in the future.